lib/compiler/scope.js

```diff
@@ -110,6 +110,12 @@ class Scope {
         this.put(name, symbol);
         return symbol;
     }
+    define_native_proc(name, args, exported) {
+        var runtimeVar = this.alloc_var(name);
+        var symbol = { type: 'native_proc', exported: exported, uname: runtimeVar, args: args };
```

Comment:
It may make sense to do something like `type: 'proc', native: true`. In the future, we’ll probably also want native modules to export consts, functions, and reducers and we certainly won’t want to introduce new types like `native_const`, just a flag so that later stages can treat them differently if necessary.